packages/calling/src/Contacts/ContactsClient.ts

```diff
@@ -272,16 +273,15 @@ export class ContactsClient implements IContacts {
         const filteredContact = inputList.Resources.filter((item) => item.id === contactList[n])[0];
 
         const {displayName, emails, phoneNumbers, photos} = filteredContact;
-        const {sipAddresses} = filteredContact[SCIM_WEBEXIDENTITY_USER];
-        const firstName = filteredContact.name.givenName;
-        const lastName = filteredContact.name.familyName;
-        const manager = filteredContact[SCIM_ENTERPRISE_USER].manager.displayName;
-        const department = filteredContact[SCIM_ENTERPRISE_USER].department;
-
-        let avatarURL = '';
-        if (photos?.length) {
-          avatarURL = photos[0].value;
+        let sipAddresses;
```

Comment:
Are we using this variable anywhere else in the code? I don't seem to see it anywehere else apart from the next line.